building/target_platforms/target_platforms_for_consumers.adoc

```diff
@@ -0,0 +1,212 @@
+
+In concrete terms, a target platform can take two forms: a `target platform definition file` and a `target platform update site`. 
+
+A `target platform definition file` is a custom Eclipse file, with a `.target` suffix, for use by Eclipse's Plug-in Development Environment (PDE) tools. The file lists  the versions of installable units (IUs) to be used as dependencies, as well as the repositories from which those dependencies can be found. The contained IUs are generally features, but can also be invididual plugins.
+
+A `target platform update site` is the result of `resolving` the target platform definition file, which is the process of downloading all the IUs, and thier contained IUs, to one's local disk. It includes the full set of jars for all dependencies our project may depend on, as well as the required Eclipse metadata to act as an update site. An update site, in Eclipse, is a mechanism used to install new (or update existing) software into Eclipse. Because of this, you could probably install the entire target platform into your Eclipse using such an update site. 
+
+See these slides for more about target platforms: http://www.slideshare.net/mickaelistria/a-journey-with-target-platforms.
+
+== Where is the target platform for JBoss Tools?
+
+JBoss Tools' `target platform definition files` are stored in github:
+
+* https://github.com/jbosstools/jbosstools-target-platforms[JBoss Tools & Red Hat JBoss Developer Studio Target Platforms]
+* https://github.com/jbosstools/jbosstools-discovery[JBoss Tools Central Target Platforms]
+* https://github.com/jbosstools/jbosstools-integration-stack[Integration Stack Target Platforms]
+
+JBoss Tools' `target platform update sites` are located here, in both unpacked and packed (zipped) format. They do *not* contain sources or SDK features.
+
+* http://download.jboss.org/jbosstools/targetplatforms/ (SNAPSHOTS and milestones)
+* http://download.jboss.org/jbosstools/static/targetplatforms/ (released versions)
+
+## Exploring the JBoss Tools target platform repository
+
+Inside the JBoss Tools Target Platform repository, the two top level folders are `jbdevstudio` and `jbosstools`.  Both of these folders contain two sub-folders: `multiple` and `unified`. 
+
+### Multiple vs unified target platforms
+
+**Multiple** target platform definition files reference several different update sites, most of which are mirrors located on http://download.jboss.org/jbosstools/. Use this .target file in Eclipse, since it can include sources when built with the proper flags.
+
+**Unified** target platform definition files are generated from the multiple ones, and are then used to produce the target platform update sites. As this file references only a single mirror, it is faster to use in Eclipse but cannot include sources, since it points at the generated target platform update site, which does not include sources.
+
+### Versioning rules
+
+JBoss Tools has a semi-intuitive versioning scheme for our target platforms. 
```

Comment:
semi-intuitive is subjective, specific is objective. if you don't like the naming scheme or don't find it correct/intuitive enough open a bug report ;)